velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -19,6 +19,11 @@
 
 namespace facebook::velox::exec::test {
 
+class QueryRunnerContext {
+ public:
+  std::unordered_map<core::PlanNodeId, std::vector<std::string>> windowFrames_;
```

Comment:
You could use a pair or struct with 2 fields for windowFrames_ as you are always guaranteed to have 2 of them.

Reply:
This is fine.